.github/workflows/publish-bundle.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish bundle to CharmHub
```

Comment:
suggestion(non-blocking): I think release is more appropriate wording than publish, given also that the command run is `charmcraft release`.

Reply:
goes for every place in this file.